types/simulations.ts

```diff
@@ -31,8 +31,10 @@ export interface ReceiveAchPaymentSimulation {
     }
     relationships: {
         account: {
-            type: "depositAccount"
-            id: string
+            data: {
+                type: "depositAccount"
+                id: string
+            }
```

Comment:
I just wrapped it in a `data` object because [that's what the docs say to do](https://docs.unit.co/simulations/#simulate-receive-ach-payment) and the Unit sandbox was throwing an error when I didn't do this.